rllib/optimizers/sync_replay_optimizer.py

```diff
@@ -25,24 +26,26 @@ class SyncReplayOptimizer(PolicyOptimizer):
     "td_error" array in the info return of compute_gradients(). This error
     term will be used for sample prioritization."""
 
-    def __init__(self,
-                 workers,
-                 learning_starts=1000,
-                 buffer_size=10000,
-                 prioritized_replay=True,
-                 prioritized_replay_alpha=0.6,
-                 prioritized_replay_beta=0.4,
-                 prioritized_replay_eps=1e-6,
-                 schedule_max_timesteps=100000,
-                 beta_annealing_fraction=0.2,
-                 final_prioritized_replay_beta=0.4,
-                 train_batch_size=32,
-                 sample_batch_size=4,
-                 before_learn_on_batch=None,
-                 synchronize_sampling=False):
+    def __init__(
+            self,
+            workers,
+            learning_starts=1000,
+            buffer_size=10000,
+            prioritized_replay=True,
+            prioritized_replay_alpha=0.6,
+            prioritized_replay_beta=0.4,
+            prioritized_replay_eps=1e-6,
+            schedule_max_timesteps=None,  # DEPRECATED
+            beta_annealing_fraction=None,  # DEPRECATED
```

Comment:
This isn't a public API so you can just change these without deprecation.

Reply:
ok.